### PR TITLE
feat: print the sequence of fibonacci and fix expected data of inverted triangle

### PR DESCRIPTION
**Pull request by nammon72**
1. What did you do? (mandatory)
- print the sequence of fibonacci
- fix the expected data of the inverted triangle
2. Why would you do that? (mandatory)
- Data homogenization of test cases
3. An explanation about your `solution or decision or choise` (if needed)

4. Checklist (if needed)

5. Action need to do before merge (if needed)

6. Action need to do after merge (if needed)

7. Others note (if needed)

### CODE REVIEW

File: src/quadratic-equation.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function solveQuadraticEquation(a, b, c) {
+  let result1 = 0;
+  let result2 = 0;
+  const delta = b * b - 4 * a * c;
```
Comment (anhdiepmmk) [inline src/quadratic-equation.js:4]: b * b should be Math.pow(b, 2)

File: src/print-the-sequence-of-fibonacci.test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const fibonacci = require('./print-the-sequence-of-fibonacci');
```
Comment (anhdiepmmk) [inline src/print-the-sequence-of-fibonacci.test.js:1]: add test case for given number = 0, 1, 2, 3

File: src/print-the-sequence-of-fibonacci.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+function getFibonacci(number) {
```
Comment (anhdiepmmk) [inline src/print-the-sequence-of-fibonacci.js:1]: getFibonacciSequences

File: src/reverse-numbers.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+function reverseNumbers(numbers) {
```
Comment (anhdiepmmk) [inline src/reverse-numbers.js:1]: delele this due to miss understand requirement

File: src/reverse-numbers.test.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+const reverse = require('./reverse-numbers');
```
Comment (anhdiepmmk) [inline src/reverse-numbers.test.js:1]: delele this due to miss understand requirement

**Review by anhdiepmmk — CHANGES_REQUESTED** [review]

File: src/print-the-sequence-of-fibonacci.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  const newNumbers = [];
```
[the remark this answers is not in the document]
Comment (nammon72) [reply src/print-the-sequence-of-fibonacci.js:2]: okay, i fixed it.

File: src/reverse-numbers.js (hunk 4 above)
Comment (nammon72) [reply src/reverse-numbers.js]: okay, I did.

File: src/reverse-numbers.test.js (hunk 5 above)
Comment (nammon72) [reply src/reverse-numbers.test.js]: okay, I did.

File: src/print-the-sequence-of-fibonacci.js (hunk 3 above)
Comment (anhdiepmmk) [reply src/print-the-sequence-of-fibonacci.js:1]: where is the new code ? can you push this ?

**Pull request merged**